README.md

```diff
@@ -1,63 +1,160 @@
+1. You'll need to have node.js installed to run network dapps. To make sure you have it installed, run `node -v`. If no version is returned, see [Nodejs.org/downloads](https://nodejs.org/en/download/).
 
-## Deployment
+2. Clone the repo with `git clone https://github.com/axelarnetwork/axelar-local-gmp-examples.git` and `cd axelar-local-gmp-examples`.
 
-Some examples need to first be deployed on the network you wish to run them on. To do this simply run `node scripts/deploy examples/${name} ${local|testnet}`. Replace `${var}` appropriately. Cannot be run for the `sent-token` and `deposit-address` examples as no deployment is needed.
+3. Build contracts and tests (The update and install takes a few minutes.):
+   ```
+   npm update && npm install
+   npm run build
+   ```
 
-## Testing
+3. To connect to the axelar network, run `npm install axelarnetwork/axelar-local-dev`.
 
-All examples can be tested with `node scripts/test/{example-name} ${local|testnet} ...params`. See below for what values params can take. All params are optional and have default values and can be ommited.
+4. To run a local node, open `axelar-local-gmp-examples` in a separate terminal and run `node scripts/createLocal`. You’ll need to have this node running to deploy the dapps.
 
-### Sent token
+   The five supported networks are deployed: Moonbeam (ChainID: 2500), Avalanche (chainID: 2501), Fantom (chainID: 2502), Ethereum (chainID: 2503), and Polygon (chainID: 2504) on port 8500.
 
-Run `node scripts/test examples/send-token ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+5. To make sure that the address we use for examples is funded on all five supported testnets, run `node scripts/checkBalances`. We use `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` to deploy and test all examples. Alternatively, you can use `--address [<address>]` to specify a different address.
 
-### Deposit Address
+## Deploy and test each example
 
-Run `node scripts/test examples/deposti-address ${local|testnet} ${source-chain} ${destination-chain} ${amount}` to send aUSDC from the source to the destination. To run on testnet you need to fund `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb` with aUSDC and replace `local` with `testnet`.
+For each example, you can allow the defaults to run or enter:
+
+- `local` or `testnet` for both the test and the deploy steps, except for deposit-address and send-token, which run on testnet.
+
+- `source-chain` and `destination-chain`
+
+- variables such as `message`, `amount`, and `account`
```

Comment:
@ggutoski Data types and defaults:
- Message: We've already established that [<message>] is a string data type, but what is the default (the original writer says there are defaults for all variables). 
- Amount, I would assume is an integer. What is the default? 
- What about account? Is that a wallet name? A string? What is the default?

Reply:
Looking at the code it does in fact seem that most (all?) arguments actually have default values.

- Based on `call-contract` I see that `message` has a default value of the form `Hello Polygon from Avalanche, it is 11:46:18 AM` https://github.com/axelarnetwork/axelar-local-gmp-examples/blob/db5ac04ddc7202d344c2730ebb8a05b972dbc5d7/examples/call-contract/index.js#L25 I don't understand your comment on `[]` being a string.
- Based on `call-contract-with-token` it seems that `amount` can be an integer or float.  Any non-integer is rounded down to the nearest integer.  https://github.com/axelarnetwork/axelar-local-gmp-examples/blob/db5ac04ddc7202d344c2730ebb8a05b972dbc5d7/examples/call-contract-with-token/index.js#L23 The default here is 10.  (The multiplication by `10e6` in the code is because amounts are internally represented in atomic units---one millionth of a token.)
- `account` is an address of the form `0xBa86A5719722B02a5D5e388999C25f3333c7A9fb`